include/AutomatableModelView.h

```diff
@@ -69,12 +69,16 @@ class LMMS_EXPORT AutomatableModelView : public ModelView
 
 	void addDefaultActions( QMenu* menu );
 
+	void setConversionFactor( float factor );
+	float getConversionFactor();
+
 
 protected:
 	virtual void mousePressEvent( QMouseEvent* event );
 
 	QString m_description;
 	QString m_unit;
+	float   m_conversionFactor;  // Factor to be applied when the m_model->value is displayed
```

Comment:
I think you might remove some whitespaces, though not necessary.
```suggestion
	float m_conversionFactor; // Factor to be applied when the m_model->value is displayed
```

Reply:
I'm also not sure which class should hold this property(model, or view?). This question is equivalent to: "Does 1.0 for the mixer volume always mean 100 for end-users?". I'm guessing it's preferable to keep stuff as-is, but I'm asking for other opinions.